Sources/Client/Model/Channel/Channel+Requests.swift

```diff
@@ -145,24 +151,27 @@ public extension Channel {
     
     @discardableResult
     internal func keystroke(client: Client,
-                            timerType: Timer.Type,
+                            timerType: Timer.Type = DefaultTimer.self,
```

Comment:
This is the internal method and it's only for tests. In the same public method explicitly is using `DefaultTimer.self`, so it's useless here.

Reply:
Useless is a strong word. Unnecessary? Yes :)

Reply:
I'm sorry. Yes, unnecessary!